modules/tinymce/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html),
 and is generated by [Changie](https://github.com/miniscruff/changie).
 
+## Unreleased
```

Comment:
We shouldn't add this to the changelog now since we now generate the changelogs using changie. So add a new changelog items using that instead of modifying this file directly.

- [ ] Use changie to add a new changelog item

Reply:
done